lib/searchable_select.ex

```diff
@@ -47,6 +47,8 @@ defmodule SearchableSelect do
     socket =
       socket
       |> assign(:search, "")
+      |> assign(:disabled, assigns[:disabled] || false)
+      |> assign(:placeholder, assigns[:placeholder] || "Search")
```

Comment:
No need for defaults at all here methinks - on 1st render they were already defined as part of socket.assigns, and defaults were given too (lines 64 & 74). Tested this and seems to work right for me:
```suggestion
      |> assign(:disabled, assigns[:disabled])
      |> assign(:placeholder, assigns[:placeholder])
```

Reply:
Agreed